hack/util.sh

```diff
@@ -96,3 +96,30 @@ function util::wait-for-condition() {
   fi
 }
 readonly -f util::wait-for-condition
+
+# util::stop-container stops a running docker container
+#
+# Arguments:
+#  - 1: container name for quering docker.
+function util::stop-container() {
```

Comment:
As we already put `docker start/stop` in the Go code, do we still need to add this in the script?

Reply:
We do not ned it for my code, but I figured it could be repurposed in case we need it. Do you think we should get rid of it?